app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
This should be inherited from usual Exception class, read docs about BaseException please
```suggestion
class NotWearingMaskError(Exception):
```